nixos/doc/manual/configuration/x-windows.chapter.md

```diff
@@ -116,48 +116,37 @@ using lightdm for a user `alice`:
 
 ## Intel Graphics drivers {#sec-x11--graphics-cards-intel}
 
-There are two choices for Intel Graphics drivers in X.org: `modesetting`
-(included in the xorg-server itself) and `intel` (provided by the
-package xf86-video-intel).
-
-The default and recommended is `modesetting`. It is a generic driver
-which uses the kernel [mode
-setting](https://en.wikipedia.org/wiki/Mode_setting) (KMS) mechanism. It
+The default and recommended driver for Intel Graphics in X.org is `modesetting`
+(included in the xorg-server package itself).
+This is a generic driver which uses the kernel [mode
+setting](https://en.wikipedia.org/wiki/Mode_setting) (KMS) mechanism, it
 supports Glamor (2D graphics acceleration via OpenGL) and is actively
-maintained but may perform worse in some cases (like in old chipsets).
-
-The second driver, `intel`, is specific to Intel GPUs, but not
-recommended by most distributions: it lacks several modern features (for
-example, it doesn't support Glamor) and the package hasn't been
-officially updated since 2015.
-
-The results vary depending on the hardware, so you may have to try both
-drivers. Use the option
-[](#opt-services.xserver.videoDrivers)
-to set one. The recommended configuration for modern systems is:
+maintained, however it still lacks Vsync and may perform worse in some cases
```

Comment:
It doesn't really "lack vsync", it just doesn't do the stupid triple buffering without an external compositor.

Reply:
Can you please stop being so dismissive about other people desktop preferences? It's starting to becoming a little annoying.

Reply:
My point is, the DDX itself doing vsync is basically always suboptimal, so we should be telling people to use a compositor, and this wording makes it sound like you just can't have vsync, even with one.

Reply:
I've mentioned that you will need a compositor in the "note" block below.